feat: :sparkles: post on running the two GitHub workshops (#185)
seedcase-project/community: posts/workshop-github-intro/index.qmd

---
title: "Developed and ran a new workshop on how to use GitHub"

> description: "We received several requests for running a workshop on using GitHub from collaborators and colleagues. So we developed a gentle introduction to using GitHub as part of our efforts to build up digitally technical capacity and upskill researchers to more effectively collaborate together :tada:"

Author: Good point!

author:
- Luke W. Johnston
date: "2025-12-16"
categories:
  - workshop
  - training
  - github
---

> Early in 2025 we received several requests at Steno Aarhus to run a

Reviewer: ```suggestion
Early in 2025 we received several requests at Steno Aarhus from colleagues and collaborators to run a
```

workshop on how to use GitHub for managing and working with files. Steno
Aarhus has been making increasing efforts to integrate use of GitHub
into the organisation overall and into individual research projects. But
many researchers and staff were still unfamiliar with how to use GitHub
effectively. So in early 2025, we started developing a [gentle

> introduction workshop to using

Reviewer: ```suggestion
introductory workshop to using
```

GitHub](https://github-intro.rostools.org).

We finished developing the material and ran the workshop on May 26th,
2025 in-person at Steno Aarhus for staff and researchers. We had 24
participants register and attend the workshop, which we didn't expect to
be so high! The workshop was very well received, with very positive
feedback from participants (see below for some plots of the feedback).

We are collaborating with the [DP-Next
project](https://dp-next.github.io/) to develop and build up their data
infrastructure, collection processes, and storage capacity. But we're

> also aiming to build up the technical expertise of their members. So

Reviewer: Same with "members" in the rest of the post.

when DP-Next officially started in September 2025, we planned to run the
workshop for their members as well.

Since they are spread across all the Steno centers in Denmark, Faroe
Islands, and Greenland, we had to run the workshop online. However this
was the first time we've ever ran a workshop online, so we had to
investigate different options and ways for doing this effectively. We
ended up using [Gather](https://gather.town/) to virtually host the
workshop.

So on September 9th, 2025, we ran the workshop online for DP-Next
members. We had 11 participants, who were a mix of PhD students and
postdocs from each of the Steno centers. As with the in-person workshop,

> participants gave very positive feedback and especially enjoyed the

Reviewer: ```suggestion
participants gave very positive feedback and they also enjoyed the
```
Just so it doesn't sound like they enjoyed the online platform more than the content 😉

online platform.

## Participant demographics

To give some "harder" numbers, the figure below shows some basic
demographics about the participants of the two workshops. The first
workshop had a large majority of participants who identified as

> "she/her" as well as different types of positions.

Reviewer: ```suggestion
workshop most participants used
she/her pronouns with diverse formal positions. In the second workshop, the pronoun distribution of she/her and he/him were more even with almost half of the participants being phD students. No participants preferred other pronouns to she/her and he/him. 
```
Not sure about the last sentence. Just wanted to make it clear that there was other choices available than she/her and he/him.

::: column-page
![Some basic demographics of the participants who registered for the two
workshops.](overview.svg){#fig-demographics
fig-alt="A series of bar plots showing an overview of who the participants were, such as their preferred pronoun and their position."}
:::

## Quantitative feedback

While we collect anonymous [qualitative
feedback](https://github.com/rostools/samwise/tree/main/inst/extdata/surveys/gh_intro)

> from participants after each workshop from a short survey, we also

Reviewer: ```suggestion
via free text fields in surveys from participants after each workshop from a short survey, we also
```
Is that a thing in English? Or is that Danglish?

Author: No, but maybe not the best written :stuck_out_tongue: 

I also modified a bit what you put.
